docs/api-ref.md

```diff
@@ -4700,21 +4700,221 @@ See [ViewItem class](#viewitem-class)
 
 
 ---
+## Webhooks
+<br>
+<br>
+Using the Tableau Server Client (TSC), you can create a new webhook, get a list of all the Webhooks, get details about a specific webhook, or delete a webhook. 
```

Comment:
Let's check for consistency on Webhooks vs webhooks. I think to match the rest of the page, we'd want to use the lower-case version unless talking about a specific class or something.